### PR TITLE
python code

### PR DESCRIPTION
**Pull request by aibrahim89**

### CODE REVIEW

File: python/test.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (MostafaAshrySOPC) [inline python/test.py:2]: please delete this line

File: python/test.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+print("This is a test")
```
Comment (MostafaAshrySOPC) [inline python/test.py:3]: where is am main func?

**Review by MostafaAshrySOPC — CHANGES_REQUESTED** [review]

File: python/test.py (hunk 1 above)
Comment (aibrahim89) [reply python/test.py:2]: i will do!

File: python/test.py (hunk 2 above)
Comment (aibrahim89) [reply python/test.py:3]: done!

**Review by MostafaAshrySOPC — APPROVED** [review]

**Pull request merged**